arangod/RocksDBEngine/RocksDBCollection.h

```diff
@@ -40,7 +41,7 @@ class Manager;
 }  // namespace cache
 
 class LogicalCollection;
-class RocksDBPrimaryIndex;
+class ManagedDocumentResult;
```

Comment:
I removed ManagedDocumentResult from our entire codebase a few weeks ago. We shouldn't bring it back.
```suggestion
```